x-pack/plugins/cases/public/components/use_push_to_service/index.tsx

```diff
@@ -185,11 +176,15 @@ export const usePushToService = ({
   const objToReturn = useMemo(
     () => ({
       pushButton:
-        errorsMsg.length > 0 ? (
+        errorsMsg.length > 0 || !hasDataToPush ? (
           <EuiToolTip
             position="top"
-            title={errorsMsg[0].title}
-            content={<p>{errorsMsg[0].description}</p>}
+            title={
```

Comment:
When and where is this tooltip shown? I don't think it should contain a button as a button in a tooltip can not be clicked as far as I know.

Reply:
<img width="360" alt="1" src="https://user-images.githubusercontent.com/6935300/123639187-2bf2aa80-d7dd-11eb-867f-3b1aba38990a.png">
the tooltip wraps the button when it is disabled

Reply:
it can still be clicked. as you can see, the button is not inside of the tooltip, but the tooltip jsx does need to surround the button in order to capture the hover